pages/games/[hash].js

```diff
@@ -97,113 +150,50 @@ function Game({ data, playerHash }) {
                     </div>
                 </div>
             )}
-            <h1 className="text-2xl">
-                Partie <span className="game-hash">{data.game}</span>
-            </h1>
-            {isCurrentPlayer ? (
-                <h2 className="text-green-500 font-bold uppercase">
-                    A moi de jouer
-                </h2>
-            ) : (
-                <h2 className="text-orange-500 font-bold uppercase">
-                    En attente de l'adversaire
-                </h2>
-            )}
-            <div className="flex items-center justify-between my-2">
-                {visibleBoard === PLAYER ? (
-                    <h2>Ma flotte</h2>
+            <div className="flex flex-col items-center p-5">
+                <h1 className="text-2xl">
+                    Partie <span className="game-hash">{data.game}</span>
+                </h1>
+                {isCurrentPlayer ? (
+                    <h2 className="text-green-500 font-bold uppercase">
+                        A vous de jouer !
+                    </h2>
                 ) : (
-                    <h2>Flotte Adverse</h2>
+                    <h2 className="text-orange-500 font-bold uppercase">
+                        En attente de l'adversaire
```

Comment:
```suggestion
                        En attente de votre adversaire
```

Just to use the same way to say things